diff --git a/envpool/entry.py b/envpool/entry.py
index f21e93c..08813de 100644
--- a/envpool/entry.py
+++ b/envpool/entry.py
@@ -20,3 +20,4 @@ import envpool.mujoco.dmc.registration  # noqa: F401
 import envpool.mujoco.gym.registration  # noqa: F401
 import envpool.toy_text.registration  # noqa: F401
 import envpool.vizdoom.registration  # noqa: F401
+import envpool.minesweeper.registration  # noqa: F401
